src/test/debuginfo/pretty-std.rs

```diff
@@ -38,6 +38,9 @@
 // gdbg-check:$6 = None
 // gdbr-check:$6 = core::option::Option::None
 
+// gdb-command: print some_string
+// gdbr-check:$7 = Some = {"IAMA optional string!"}
```

Comment:
We should try making this `gdb-check` instead of just `gdbr-check`, so both kinds of GDB are tested.